client/packages/system/src/Report/api/hooks/useReport.ts

```diff
@@ -6,7 +6,7 @@ import { REPORT } from './keys';
 export const useReport = (id: string) => {
   const { reportApi, storeId } = useReportGraphQL();
 
-  const queryKey = [REPORT, storeId];
+  const queryKey = [REPORT, storeId, id];
```

Comment:
👀  fix for the bug... cache wasn't getting cleared so reports needed double clicks to get the right one